src/get-started/new-typography-scale/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: layout-pane.njk
+title: Updating your service to use the new typography scale
+section: Get started
+theme: How to guides
+order: 6
+description: The GOV.UK Design System team have been working on updating our typography scale provided with GOV.UK Frontend. This new scale is now available to use and will be enforced by default in GOV.UK Frontend version 6.0.0. Here's how to use it, what's changed and what to expect when implementing it into your service.
```

Comment:
@claireashworth might have a better suggestion but I think we can clear this up a bit...

```suggestion
description: The GOV.UK Design System team have been working on updating the typography scale. This new scale is now available to use. You currently need to opt into it using a feature flag, but it will be switched on by default in GOV.UK Frontend version 6.0.0. Here's how to use it, what's changed, and what to expect when implementing it into your service.
```

Reply:
Another potential option for this section. Also, do we want to change all instances of typography scale to type scale?

```suggestion
description: The GOV.UK Design System team has updated the GOV.UK Frontend typography scale. This new scale is an available option to use now, but will be enforced by default in GOV.UK Frontend version 6.0.0. Here's how to use it, what's changed and what to expect when implementing it into your service.
```

Reply:
Yeah good point, I think it should say 'type scale' everywhere

Reply:
```suggestion
description: The GOV.UK Design System team has updated the GOV.UK Frontend type scale. This new scale is an available option to use now, but will be enforced by default in GOV.UK Frontend version 6.0.0. Here's how to use it, what's changed and what to expect when implementing it into your service.
```